src/core/server/base.js

```diff
@@ -65,6 +66,9 @@ export default function(routes, createStore) {
 
       const store = createStore();
 
+      store.dispatch({type: 'SET_JWT', payload: {token: cookie.load('jwt_api_auth_token')}});
```

Comment:
As an example the disco pane won't need auth - as long as having it in the base won't get in the way that's fine.